torch/_torch_docs.py

```diff
@@ -4897,6 +4897,31 @@ def merge_dicts(*dicts):
     tensor([3, 2, 4])
 """.format(**common_args))
 
+add_docstr(torch.fmax, r"""
+fmax(input, other, *, out=None) -> Tensor
+
+Computes the element-wise maximum of :attr:`input` and :attr:`other`.
+
+.. note::
+    If one of the elements being compared is a NaN, then the non-nan element is returned. 
+    If both elements are NaNs then the first is returned.
```

Comment:
"then the first is returned" -> "then NaN is returned"

since we don't allow complex inputs the ordering isn't relevant